next-frontend/build-translations.js

```diff
@@ -19,6 +20,34 @@ function flattenObject(obj, prefix = "") {
   }, {});
 }
 
+// Add Country Names if not translated already
+function addCountryNames(translation, lang) {
+  const supportedLanguages = countries.getSupportedLanguages();
+  const iso639LanguageCode = lang.slice(0, 2);
+
+  if (!supportedLanguages.includes(iso639LanguageCode)) {
+    return translation;
+  }
+  const iso3166CountryCodes = Object.keys(countries.getAlpha2Codes());
+
+  // Load the language file for the country names
+  countries.registerLocale(
+    require(`i18n-iso-countries/langs/${iso639LanguageCode}.json`),
+  );
```

Comment:
I am not sure whether this `registerLocale` logic is required, because the code you're writing here is running on the server. Quote from the library README:
> Install it using npm: `npm install i18n-iso-countries`
> ```
> var countries = require("i18n-iso-countries");
> ```
> If you use i18n-iso-countries with Node.js, you are done.

Reply:
Right afterwards, the README also specifies:

> If you use the package in a browser environment, you have to register the languages you want to use [...]

but we are not using the package directly in a browser environment anyways